### PR TITLE
feat: generate types using OpenAPI

### PR DESCRIPTION
**Pull request by ilfa**

### CODE REVIEW

**Review by TheUnderScorer — APPROVED** [review]

File: src/generatedApiTypes.ts [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+          request_id?: string;
+          /** Filter events by custom identifier */
+          linked_id?: string;
+          /** Limit scanned results */
+          limit?: number;
+          /** Used to paginate results */
+          before?: number;
+        };
+      };
+      responses: {
```
Comment (makma) [inline src/generatedApiTypes.ts:25]: I'm pretty sure we return also other statuses than `200` and `429`. Shouldn't be these also included in the definition?

File: src/generatedApiTypes.ts [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+export interface paths {
+  '/visitors/{visitor_id}': {
+    /** This endpoint allows you to get a history of visits with all available information. Use the visitorID as a URL path parameter. This API method is scoped to a visitor, i.e. all returned information is by visitorId. */
+    get: {
+      parameters: {
+        path: {
+          visitor_id: string;
+        };
+        query: {
```
Comment (makma) [inline src/generatedApiTypes.ts:14]: What about api key query param? Why it's not included? src: https://dev.fingerprint.com/docs/server-api#api-key-query-parameter

Comment (ilfa) [reply src/generatedApiTypes.ts:14]: It [is described in the security section](https://github.com/fingerprintjs/fingerprint-pro-server-api-openapi/blob/main/schemes/fingerprint-server-api.yaml#L83) of the schema, but codegen ignores it.

File: src/generatedApiTypes.ts (hunk 1 above)
Comment (ilfa) [reply src/generatedApiTypes.ts:25]: Yes, we should.

**Review by makma — APPROVED** [review]

**Pull request merged**